jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -63,7 +63,7 @@ public static JibBuildRunner createJibBuildRunnerForDockerDaemonImage(
       InferredAuthProvider inferredAuthProvider,
       ProjectProperties projectProperties,
       @Nullable Path dockerExecutable,
-      @Nullable Map<String, String> dockerEnvironment,
+      Map<String, String> dockerEnvironment,
```

Comment:
Since `rawConfiguration` now holds `dockerExecutable` and `dockerEnvironment`, we don't need to pass them? I think it's good for overall consistency.